zubbi/scraper/repo_parser.py

```diff
@@ -91,6 +91,7 @@ def parse_job_definitions(self, file_path, job_info):
                         doc = render_sphinx(job.description)
                         job.description_html = doc["html"]
                         job.platforms = doc["platforms"]
+                        job.reusable = doc["reusable"]
```

Comment:
The same should also be done for roles, or? This seems to be missing in this PR (or I cannot find it).

Reply:
For roles, it's already covered here: https://github.com/bmwcarit/zubbi/blob/4a5665de93cc7108bdd67b1fc36af43af918240f/zubbi/scraper/repo_parser.py#L137